tests/integration/hive_metastore/test_migrate.py

```diff
@@ -671,3 +671,45 @@ def test_migrate_external_tables_with_spn_azure(
             match = True
             break
     assert match
+
+
+def test_migrate_table_uppercase_column_names(ws,
+                                              sql_backend,
+                                              runtime_ctx,
+                                              make_catalog,
+                                              make_mounted_location,
+                                              make_random):
+    table_name = f"ucx_{make_random(4)}".lower()
```

Comment:
why `lower()` when we want to check table names using uppercase ?

Reply:
The bug is for checking the column names being converted to lower case.